internal/provider/resource_cert_request.go

```diff
@@ -193,6 +193,13 @@ func (r *certRequestResource) Schema(_ context.Context, req resource.SchemaReque
 							},
 							Description: "Distinguished name: `SERIALNUMBER`",
 						},
+						"email_address": schema.StringAttribute{
+							Optional: true,
+							PlanModifiers: []planmodifier.String{
+								stringplanmodifier.RequiresReplace(),
+							},
+							Description: "Email Address: `1.2.840.113549.1.9.1`",
```

Comment:
Just a small change to be a little more specific what the value is
```suggestion
							Description: "ASN.1 Object Identifier (OID): `1.2.840.113549.1.9.1`",
```